climada/hazard/test/test_base.py

```diff
@@ -1022,6 +1022,25 @@ def test_ref_all_pass(self):
         self.assertAlmostEqual(inten_stats[1][66], 70.608592953031405)
         self.assertAlmostEqual(inten_stats[3][33], 88.510983305123631)
         self.assertAlmostEqual(inten_stats[2][99], 79.717518054203623)
+        
+    def test_local_return_period(self):
+        """Compare local return periods against reference."""
+        haz = dummy_hazard()
+        haz.intensity = sparse.csr_matrix([[1.1, 2.1, 3.1],
+                                   [0.1, 3.1, 2.1],
+                                   [4.1, 1.1, 0.1],
+                                   [2.1, 3.1, 3.1]])
+        haz.frequency = np.full(4, .5)
+        threshold_intensities = np.array([1., 2., 4.])
+        return_stats = haz.local_return_period(threshold_intensities)
+        np.testing.assert_allclose(
+            return_stats,
+            np.array([
+                [0.66666667, 0.5, 0.66666667],
+                [1., 0.66666667, 0.66666667],
+                [2., np.nan, np.nan]
+                ])
+        )
```

Comment:
Can you make a simpler test? Why use numbers that are hard to compute instead of a simple set of numbers? Ideally, the test should be immediately clear. Here one needs to take a piece of paper and compute before knowing what is going on.

Also, you should test the method `_local_return_period`. 

Also, you should test that the chunking is working as intended.

Reply:
I simplified the numbers in the test. 
I guess the test is a test of both local_return_period and _loc_return_period? Also, I don't know how to test the chunking with a "simple" test? Then I would need to use a hazard large enough that the chunking is activated? Feedback welcome :)